subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -29,6 +29,7 @@ LOG_MODULE_REGISTER(nrf_cloud_codec, CONFIG_NRF_CLOUD_LOG_LEVEL);
 #define PAIRED_STR "paired"
 
 #define RSRP_ADJ(rsrp) (rsrp - ((rsrp <= 0) ? 140 : 141))
+#define RSRQ_ADJ(rsrq) (((double)rsrq * 0.5) - 19.5)
```

Comment:
this and the conversion above it are a bit mysterious -- perhaps some comments about that the scaling and shifting is doing?